src/pool.h

```diff
@@ -32,11 +32,11 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef POOL_H
 #define POOL_H
 #define MAX_RIG_ID 32
+#define MAX_RIGS 1024
```

Comment:
If the function took start/end pointers, MAX_RIGS can be removed and MAX_RIG_ID can go in the c file.